packages/atomic/src/components/result-template-components/atomic-result-printable-uri/atomic-result-printable-uri.tsx

```diff
@@ -50,6 +51,9 @@ export class AtomicResultPrintableUri {
    */
   @Prop() target = '_self';
 
+  @FocusTarget()
+  private expandedPartFocus!: FocusTargetController;
```

Comment:
Hum.... since the focus can only be in one place at a time in a search page, and that this controller allows to jump to many different place in the page (it's not only local navigation, correct me if I'm wrong), would it then make sense to start thinking this should instead be similar to our other bindings ?

Like i18n, or the engine, etc.
A single instance available to all components in the interface ?

Reply:
Intuitively yes, but this component generally (and especially in this case) waits for the re-render of the component it's attached to. Additionally, in some cases we need multiple instances of this controller to manage different focus "events" and their own expected targets.